proto/rpc.proto

```diff
@@ -6,6 +6,7 @@ import "requests.proto";
 import "responses.proto";
 
 service Api {
+    rpc CheckNullifiersByPrefix(requests.CheckNullifiersByPrefixRequest) returns (responses.CheckNullifiersByPrefixResponse) {}
```

Comment:
Nit: this should move one down to be properly sorted alphabetically (shorter string comes 1st).